packages/garbo-lib/src/wanderer/guzzlr.ts

```diff
@@ -94,7 +94,7 @@ export function guzzlrFactory(
   locationSkiplist: Location[],
   options: WandererFactoryOptions,
 ): WandererTarget[] {
-  if (Guzzlr.have()) {
+  if (Guzzlr.have() && _type !== "freerun") {
```

Comment:
If we're actually using the variable it shouldn't be preceded by an underscore anymore